src/instructlab/sdg/llmblock.py

```diff
@@ -148,10 +155,9 @@ def _generate(self, samples, **gen_kwargs) -> list:
             )
             return [choice.text.strip() for choice in response.choices]
 
-        n = gen_kwargs.get("n", 1)
         results = []
         for prompt in prompts:
-            for _ in range(n):
+            for _ in range(generate_args["n"]):
```

Comment:
Why is this not `get_n()` ?

Reply:
That would work and make the code more clear, but in this particular code path, the value was already converted up several lines when it did

```python
    generate_args = self._gen_kwargs(**gen_kwargs)
```

Reply:
pro: always use `self._get_n()` so nobody is trying to think of when to use it or not

con: technically not required here 

shrug